### PR TITLE
feat: mempool lite

### PR DESCRIPTION
**Pull request by tchardin**
Solves #3 & #6

### CODE REVIEW

File: browser/src/impl.ts [hunk 1]
```diff
@@ -31,73 +40,209 @@ export type ChainInfo = {
...
+      appSpecificWeight: 1,
+      IPColocationFactorWeight: -100,
+      IPColocationFactorThreshold: 5,
+      IPColocationFactorWhitelist: new Set(),
+      behaviourPenaltyWeight: -10.0,
+      behaviourPenaltyThreshold: 6,
+      behaviourPenaltyDecay: 0.998722,
+      decayInterval: 1000.0,
+      decayToZero: 0.1,
+      retainScore: 3600 * 6,
```
Comment (tchardin) [inline browser/src/impl.ts]: @alexander-camuto here are all the params. Pretty much mirrored everything in [lotus](https://github.com/filecoin-project/lotus/blob/c602412eba07c3e5650b6ebd59d93f640411ee1b/node/modules/lp2p/pubsub.go#L76). I don't quite understand every param yet.

Comment (alexander-camuto) [reply browser/src/impl.ts]: I don't think these will be our salvation in terms of preventing the browser peer from getting booted, as these just pertain to scoring the _other_ peers we interact with. We'll maybe have to play around with some of the other `GossipsubOpts` ?

**Comment by alexander-camuto** [comment]
Getting a few issues when trying to build the staging app with the instructions in the README. Required me running `npm install` from within `browser/app` to get it to install all packages. Here's a sample of the errors if I follow the current README steps: 

```
✘ [ERROR] Could not resolve "react"

    Spinner.tsx:1:23:
      1 │ import * as React from "react";
        ╵                        ~~~~~~~

  You can mark the path "react" as external to exclude it from the bundle, which will remove this error.


✘ [ERROR] Could not resolve "react-dom/client"

    index.tsx:2:26:
      2 │ import * as ReactDOM from "react-dom/client";
        ╵                           ~~~~~~~~~~~~~~~~~~

  You can mark the path "react-dom/client" as external to exclude it from the bundle, which will remove this error.

2 out of 8 errors shown

```

**Comment by geomatrick** [comment]
image

I'm getting the issue when trying to connect to the network.

Steps:

1. install npm packages in both browser folder and browser/app folder
2. Run `npm run build` in browser folder
3. Run `npm run start` in app folder
4. input multiaddr and network name into network input boxes.
5. clicked connect

**Comment by alexander-camuto** [comment]
> image
> 
> I'm getting the issue when trying to connect to the network.
> 
> Steps:
> 
>     1. install npm packages in both browser folder and browser/app folder
> 
>     2. Run `npm run build` in browser folder
> 
>     3. Run `npm run start` in app folder
> 
>     4. input multiaddr and network name into network input boxes.
> 
>     5. clicked connect

Tried debugging with @patrickwoodhead to see if other locally running processes might be causing issues (stations, Brave IPFS node, punchr, VPN etc...) but issue still persists when shutting those off.

**Comment by geomatrick** [comment]
> image
> 
> I'm getting the issue when trying to connect to the network.
> 
> Steps:
> 
> 1. install npm packages in both browser folder and browser/app folder
> 2. Run `npm run build` in browser folder
> 3. Run `npm run start` in app folder
> 4. input multiaddr and network name into network input boxes.
> 5. clicked connect

This is now fixed following a hard delete of the repo and reclone, reinstall

**Review by alexander-camuto — APPROVED** [review]

**Pull request merged**